<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATI Label Validator</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>600</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>416</width>
     <height>416</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>414</width>
      <height>414</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_imageA">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>416</width>
       <height>416</height>
      </rect>
     </property>
     <property name="text">
      <string>Image A</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>416</width>
     <height>416</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>414</width>
      <height>414</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_imageB">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>416</width>
       <height>416</height>
      </rect>
     </property>
     <property name="text">
      <string>Image B</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1051</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_openB">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_openA">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_folderB">
      <property name="placeholderText">
       <string>set the second folder containing label .json files</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Folder B</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Folder A</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_folderA">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>set the first folder containing label .json files</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="3">
     <widget class="QPushButton" name="pushButton_process">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Process</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>938</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1006</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
